src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -38,6 +40,8 @@ public class ViewerApiController : ApiController
         private static readonly Common.Config.GlobalConfiguration globalConfiguration = new Common.Config.GlobalConfiguration();
 
         private static readonly string cachePath = Path.Combine(globalConfiguration.Viewer.GetFilesDirectory(), globalConfiguration.Viewer.GetCacheFolderName());
+        
+        private IFileWrapper FileWrapper;
```

Comment:
Codacy found an issue: [Make 'FileWrapper' 'readonly'.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=6117782)